aiida/cmdline/commands/cmd_run.py

```diff
@@ -109,8 +108,6 @@ def run(scriptname, varargs, group, group_name, exclude, excludesubclasses, incl
             # Must add also argv[0]
             new_argv = [scriptname] + list(varargs)
             with update_environment(new_argv=new_argv):
-                # Add local folder to sys.path
-                sys.path.insert(0, os.path.abspath(os.curdir))
```

Comment:
Note: This adds the current working directory to the path - however this is done by python automatically (I checked sys.path before and after this line to check)

Reply:
For both version of Python?

Reply:
I didn't check for both versions but I'm pretty sure this is the case almost since the beginning of python.

Reply:
Fair enough, I am simply spitballing, trying to come up with a reason why it may have been added to begin with - and it seemed a reasonable explanation.